firebase/firestore/writeFirestore.js

```diff
@@ -0,0 +1,106 @@
+//Send announcement data to Firestore. Fixed 
+const WriteGeneralAnnouncement = () => {
+    const sendData = () => {
+        try {
+            firebase
+                .firestore()
+                .collection('Announcements')
+                .doc() // leave as .doc() for a random unique doc name to be assigned
+                .set({
+                    title: 'Title ',
+                    paragraph: 'some text announcement',
+                    time: firebase.firestore.FieldValue.serverTimestamp(),
+                })
+                .then(alert('Announcement was successfully added.'))
+        } catch (error) {
+            console.log(error)
+            alert(error)
+        }
+    }
+    
+    //Writes to a specific course's document
+    const WriteCourseAnnouncement = () => {
+        const sendData = () => {
+            try {
+                firebase
+                    .firestore()
+                    .collection('Announcements')
+                    .doc(CourseCode) 
+                    .set({
```

Comment:
can we have 
`title:
time:
course:
content:
`
instead? just matches and makes more sense with my admin code

Reply:
as params i mean, so content instead of paragraph and just add another field for course